docs/HOW_TO_USE_AMI.md

```diff
@@ -0,0 +1,34 @@
+## Creating a Log Export Container AWS AMI
+1. Go to the EC2 management panel and click on **Launch instances**.
+
+2. Click on the "My AMIs" tab and select your Log Export Container AMI (see [How to create an AMI](docs/HOW_TO_CREATE_AMI.md)).
+![image](https://user-images.githubusercontent.com/20745533/158184010-c5fe8fc2-afd7-4f9d-9b87-fec72a16b381.png)
+
+3. Select your Instance Type and click on **Next: Configure Instance Details**.
+
+4. In this step you need to provide the Log Export Container configuration environment variables. To do that you will need to follow the template from [ami-variables-user-data.sh.example](ami-variables-user-data.sh.example). Copy the content from the file, paste it into **Advanced Details > User data** field and modify it with your desired configurations.
+
+In the following example we're going to configure a dummy [Mongo output](docs/CONFIGURE_MONGO.md) and enable [SSH Events Decoding](docs/CONFIGURE_SSH_DECODE.md) following the template.
+![image](https://user-images.githubusercontent.com/20745533/158190367-645eac78-28cb-476d-82dd-9f0484bf76b3.png)
+
+After adding your environment variables, click on **Next: Add Storage**.
+
+5. Configure your storage or leave it with the default values and click on **Next: Add Tags**.
+
+6. Add tags if you want and click on **Next: Configure Security Group**.
+
+7. In this step you can select an existing security group (if you have already created one from the previous tutorial on [How to create an AMI](docs/HOW_TO_CREATE_AMI.md)) or you can create a new security group as follows:
+
+With the option "Create a new security group" selected, click on **Add Rule** and configure it as follows:
```

Comment:
Make a safe comment saying that you need to be careful with the SSH default config, and say that the user might want to change that to an specific IP address.